packages/cfd/src/Constants/jurisdiction-contents/jurisdiction-svg-contents.ts

```diff
@@ -79,4 +81,6 @@ export const jurisdictionSvgContents = (): TJurisdictionCardItems => ({
             description: `${localize('Deriv (SVG) LLC (company no. 273 LLC 2020)')}`,
         },
     ],
+    synthetic_verification_docs: ['not_applicable'],
```

Comment:
do we need this ['not_applicable'], wont empty array work here?

Reply:
for accessing the icon name from the constant object of file` jurisdiction-verification-contents`, this name is required.